portal/eproms_substudy_tailored_content/src/js/Utility.js

```diff
@@ -110,3 +110,23 @@ export function PromiseAllSettledPolyfill() {
     };
   }
 }
+
+export function ElementClosestPolyfill() {
+  //see https://developer.mozilla.org/en-US/docs/Web/API/Element/closest
```

Comment:
Had to leave a comment about something! :D
```suggestion
  //see https://developer.mozilla.org/en-US/docs/Web/API/Element/closest#polyfill
```

Reply:
thank you, Ivan, fixed.